docs/reference/tmx-map-format.rst

```diff
@@ -245,6 +245,23 @@ Can contain any number: :ref:`tmx-terrain`
 
 Can contain at most one: :ref:`tmx-properties`
 
+.. _tmx-tileset-transformations:
+
+<transformations>
+~~~~~~
+
+- **hflip:** Whether the tiles in this set can be flipped horizontally to
+  stand in for missing patterns (default 0)
+- **vflip:** Whether the tiles in this set can be flipped vertically to
+  stand in for missing patterns (default 0)
+- **rotate:** Whether the tiles in this set can be rotated by 90 degree to
+  stand in for missing patterns (default 0)
```

Comment:
I'm not sure about the "to stand in for missing patterns", since that seems to imply untransformed tiles are preferred.